config.py

```diff
@@ -1,6 +0,0 @@
-import os
-
-class Config(object):
-    SECRET_KEY = os.environ.get('SECRET_KEY') or 'a-secret-key'
```

Comment:
Since we should probably use an environment variable for the ffmpeg path and maybe have a DEBUG option, maybe lave in a simple config with those two? Also, is the secret key used anywhere for hashing user id's (or should we do that?).

Reply:
not really needed
IDK why we have a secret key